examples/bicycle_wheel_contact_constraint_forces/lateral_tire_forces.m

```diff
@@ -0,0 +1,69 @@
+function [Ff, Fr] = lateral_tire_forces(q, u, up, p)
+% LATERAL_TIRE_FORCES - Returns the lateral tire constraint forces of the
+% Carvallo-Whipple model.
+%
+% Syntax: [Ff, Fr] = lateral_tire_forces(q, u, up, p)
+%
+% Inputs:
+%   q - Coordinates at time t, size 2x1, [q4 (roll angle), q7 (steer angle)]
+%   u - Independent generalized speeds at time t, size 3x1 [u4 (roll rate),
+%       u6 (rear wheel rate), u7 (steer rate)]
+%   up - Derivatives of the independent generalized speeds at time t, size
+%        3x1 [u4p (roll ang acceleration), u6p (rear ang acceleration), u7p
+%        (steer ang accerlation)]
+%   p - Constant parameter structure which includes: [d1, d2, d3, g, ic11,
+%       ic22, ic31, ic33, id11, id22, ie11, ie22, ie31, ie33, if11, if22,
+%       l1, l2, l3, l4, mc, md, me, mf, rf, rr]
+% Outputs:
+%   Ff - Lateral constraint force at the front wheel contact.
+%   Fr - Lateral constraint force at the rear wheel contact.
+
+% unpack the function inputs
+q4 = q(1);  % roll angle
+q7 = q(2);  % steer angle
+
+u4 = u(1);  % roll rate
+u6 = u(2);  % rear wheel rate
+u7 = u(3);  % steer rate
+
+u4p = up(1);  % roll angular acceleration
+u6p = up(2);  % rear wheel angular acceleration
+u7p = up(3);  % steer angular acceleration
```

Comment:
For a simulation function we need to calculate these with the equations of motion. Something like:

```python
M = kane.mass_matrix
f = kane.forcing
eval_M, eval_f = MatrixGenerator(M, f)
u_independent_dot = np.linalg.solve(M, f)
u4p = u_independent_dot[0]
u6p = u_independent_dot[6]
u7p = u_independent_dot[7]
```